setup.py

```diff
@@ -1,13 +1,17 @@
 from setuptools import setup
 
+_dependencies = ['requests', 'gitpython', 'pyserial', 'python-dateutil']
+
 setup(
     name='friskby',
-    version='0.61.0',
+    version='0.61.1',
     description='The friskby module',
-    url='http://github.com/FriskbyBergen/friskby',
+    url='http://github.com/FriskbyBergen/python-friskby',
     author='Friskby Bergen',
     author_email='pgdr@statoil.com',
-    license='GPL v3',
+    license='GNU General Public License, Version 3',
     packages=['friskby'],
-    zip_safe=False
+    zip_safe=False,
+    setup_requires=_dependencies,
+    install_requires=_dependencies
```

Comment:
you need `extras_requires` with a `tests` directive including pylint. Right now there seems to be no indication that pylint would be useful for the package's tests.